### PR TITLE
fuck dna

### PR DESCRIPTION
**Pull request by spookydonut**
fuck dna its been chopped and gutted so much it has no purpose and i want to see tg/goon genetics ported

### CODE REVIEW

**Comment by psykzz** [comment]
It may be annoying but you may need to touch up on blood.dm

I have a draft PR where im trying to make xenoblood useful for reactions for stronger weapons and mutations, but having so many problems because of DNA.

Happy to see it go, but i know its going to be a problem for you.

**Comment by spookydonut** [comment]
> 
> 
> It may be annoying but you may need to touch up on blood.dm
> 
> I have a draft PR where im trying to make xenoblood useful for reactions for stronger weapons and mutations, but having so many problems because of DNA.
> 
> Happy to see it go, but i know its going to be a problem for you.

yeah i have to since blood overlays reference dna but dont really need to

File: code/game/blood.dm [hunk 1]
```diff
@@ -1,84 +1,50 @@
...
 
 
-/obj/item/add_blood(list/blood_dna, b_color)
-	if(flags_atom & NOBLOODY)
-		return 0
-	var/blood_count = blood_DNA && blood_DNA.len
-	..()
-	if(!blood_count && !blood_overlay)//apply the blood-splatter overlay if it isn't already in there
+/obj/item/add_blood(b_color)
+	if(!..())
```
Comment (DominikPanic) [inline code/game/blood.dm]:
```
. = ..()
if(!.)
```

File: code/modules/recycling/disposal.dm [hunk 2]
```diff
@@ -468,10 +468,6 @@
 	for(var/atom/movable/AM in D)
 		AM.loc = src
 		SEND_SIGNAL(AM, COMSIG_MOVABLE_DISPOSING, src, D)
-		if(ishuman(AM))
-			var/mob/living/carbon/human/H = AM
-			if(FAT in H.mutations) //Is a human and fat?
-				has_fat_guy = 1 //Set flag on holder
```
Comment (DominikPanic) [inline code/modules/recycling/disposal.dm]: This var seems to not be used elsewhere, could you remove it?

File: code/game/objects/items/weapons/swords_axes_etc.dm [hunk 3]
```diff
@@ -26,27 +26,12 @@
...
 /obj/item/weapon/classic_baton/attack(mob/living/M as mob, mob/living/user as mob)
-	if ((CLUMSY in user.mutations) && prob(50))
-		to_chat(user, "<span class='warning'>You club yourself over the head.</span>")
-		user.KnockDown(3 * force)
-		if(ishuman(user))
-			var/mob/living/carbon/human/H = user
-			H.apply_damage(2*force, BRUTE, "head")
-		else
-			user.take_limb_damage(2*force)
+	if(!..())
```
Comment (DominikPanic) [inline code/game/objects/items/weapons/swords_axes_etc.dm]:
```
. = ..()
if(!.)
```

File: code/modules/mob/living/carbon/human/human_damage.dm [hunk 4]
```diff
@@ -17,8 +17,8 @@
 
 	health = species.total_health - oxy_l - tox_l - clone_l - total_burn - total_brute
 
-	if(CONFIG_GET(flag/husking_on) && ((species.total_health - total_burn) < get_death_threshold() * 4))
-		ChangeToHusk()
+	//if(CONFIG_GET(flag/husking_on) && ((species.total_health - total_burn) < get_death_threshold() * 4))
```
Comment (DominikPanic) [inline code/modules/mob/living/carbon/human/human_damage.dm]: Commented out code

File: code/game/blood.dm (hunk 1 above)
Comment (spookydonut) [reply code/game/blood.dm]: theres a hard return true at the end of the proc.

**Comment by Rohesie** [comment]
Should have merged this one before.
Merge conflict now.

**Pull request merged**